fuzz/fuzz_targets/utils.rs

```diff
@@ -1,71 +1,48 @@
 #![allow(dead_code)]
 
 use arbitrary::{Arbitrary, Unstructured};
-use wasmi::{core::ValType, Val};
-
-pub fn exec_config() -> wasm_smith::Config {
-    wasm_smith::Config {
-        export_everything: true,
-        allow_start_export: false,
-        reference_types_enabled: false,
-        max_imports: 0,
-        max_memory32_bytes: (1 << 16) * 1_000,
-        // Note: wasmi does not support 64-bit memory, yet.
-        memory64_enabled: false,
-        max_data_segments: 10_000,
-        max_element_segments: 10_000,
-        max_exports: 10_000,
-        max_elements: 10_000,
-        min_funcs: 1,
-        max_funcs: 10_000,
-        max_globals: 10_000,
-        max_table_elements: 10_000,
-        max_values: 10_000,
-        max_instructions: 100_000,
-        exceptions_enabled: false,
-        simd_enabled: false,
-        threads_enabled: false,
-        gc_enabled: false,
-        tail_call_enabled: false,
-        ..Default::default()
-    }
+use wasmi::{core::ValType, ExternRef, FuncRef, Val};
+
+pub fn disable_unsupported_config(config: &mut wasm_smith::Config) {
+    config.gc_enabled = false;
+    config.exceptions_enabled = false;
+    config.max_memories = 1;
+    config.memory64_enabled = false;
+    config.relaxed_simd_enabled = false;
+    config.simd_enabled = false;
+    config.threads_enabled = false;
 }
 
-pub fn arbitrary_exec_module(seed: &[u8]) -> arbitrary::Result<wasm_smith::Module> {
-    let mut unstructured = Unstructured::new(seed);
-    wasm_smith::Module::new(exec_config(), &mut unstructured)
+pub fn arbitrary_config(unstructured: &mut Unstructured) -> arbitrary::Result<wasm_smith::Config> {
+    let mut config = wasm_smith::Config::arbitrary(unstructured)?;
+    disable_unsupported_config(&mut config);
+    Ok(config)
 }
 
-pub fn arbitrary_translate_module(seed: &[u8]) -> arbitrary::Result<wasm_smith::Module> {
-    let mut unstructured = Unstructured::new(seed);
-
-    let config = wasm_smith::Config::arbitrary(&mut unstructured);
-
-    config.map(|mut config| {
-        config.gc_enabled = false;
-        config.exceptions_enabled = false;
-        config.simd_enabled = false;
-        config.threads_enabled = false;
-
-        wasm_smith::Module::new(config, &mut unstructured)
-    })?
+pub fn arbitrary_swarm_config_module(
```

Comment:
Sorry for the confusion, but what exactly is a swarm config module?

Reply:
Swarm testing is a term we've lifted off of this paper

https://users.cs.utah.edu/~regehr/papers/swarm12.pdf

Using random wasm_smith configurations helps with coverage.